conan/conanfile_windows.txt

```diff
@@ -0,0 +1,10 @@
+[requires]
+gtest/1.15.0
+tinyxml2/10.0.0
+npcap/1.70
```

Comment:
Is there a reason why you chose `npcap` instead of `wincap`? Better alternative? More actively maintained? Just curious

Reply:
WinPCAP is dead: it is not maintained and it rely on the NDIS 5.0 API, which was superseed by the 6.0 version in Windows Vista.

The 5.0 API is still eanbled in the latest WIndows version as of today, but it is mark legacy and could be removed anytime (especially since it seems that there is a potential cybersecurity breach when using it).

However, npcap uses the winpcap API, so one can just link with winpcap and it should work.

So to summarize:
WinPCAP pros:
* free
* available on all systems (since Windows 98 to Windows 11)
WinPCAP cons:
* slow
* not maintained
* can be disabled anytime in the latest Windows version

NPCAP pros:
* fast
* maintained

NPCAP cons:
* not that free (but you are granted a few free licence if you do no use it for commercial purpose)